firefox-ios/Client/Frontend/Browser/ZoomPageBar.swift

```diff
@@ -206,6 +206,20 @@ class ZoomPageBar: UIView, ThemeApplicable, AlphaDimmable {
         button.setContentCompressionResistancePriority(.required, for: .horizontal)
     }
 
+    private static func setButtonInsets(_ button: UIButton, contentInsets: UIEdgeInsets) {
+        button.configuration = .plain()
+        button.configuration?.contentInsets = NSDirectionalEdgeInsets(top: contentInsets.top,
+                                                                      leading: contentInsets.left,
+                                                                      bottom: contentInsets.bottom,
+                                                                      trailing: contentInsets.right)
+    }
+
+    private static func setButtonHighlightColor(_ button: UIButton, color: UIColor) {
+        button.configuration?.imageColorTransformer = UIConfigurationColorTransformer({ [weak button] baseColor in
+            return button?.state == .highlighted ? color : baseColor
+        })
+    }
```

Comment:
The first function was static because you're using it in the initializer, so that makes sense. But is there a reason for this function to be static?

Reply:
@adudenamedruby sorry my comments are visible just now, i didn't start the review 😅